lib/jekyll/url.rb

```diff
@@ -157,5 +158,9 @@ def self.escape_path(path)
     def self.unescape_path(path)
       Addressable::URI.unencode(path.encode("utf-8"))
     end
+
+    def self.cache
```

Comment:
If this is only used for `escape_path` what do you think about calling this `escape_path_cache`?

Reply:
make sense.. especially since the cache is defined in the Singleton class, the cache would be indifferent to multiple instances of the `URL` class..

Reply:
on second thoughts.. what do you think about caching *both* escaped and unescaped paths..?
i.e. stash results from both `self.escape_path` and `self.unescape_path` in the same cache...
```ruby
cache["escaped"][path]   = ...
cache["unescaped"][path] = ...
```